File: ABC177/A-Dont_be_Late.c
#include <stdio.h>

int	main(void)
{
	int		d;
	int		t;
	int		s;
	char	*ans;

	scanf("%d", &d);
	scanf("%d", &t);
	scanf("%d", &s);
	ans = "Yes";
	if (d > s * t)
		ans = "No";
	puts(ans);
	return 0;
}

File: ABC177/A_Dont_be_late.java
import java.util.Scanner;

public class A_Dont_be_late {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		int		d, t, s;
		String	ans;

		d = sc.nextInt();
		t = sc.nextInt();
		s = sc.nextInt();
		sc.close();
		ans = "Yes";
		if (t * s < d)
			ans = "No";
		System.out.print(ans);
	}
}
